exporter/collector/logs.go

```diff
@@ -308,6 +308,15 @@ func (l logMapper) createEntries(ld plog.Logs) (map[string][]*logpb.LogEntry, er
 	return entries, errors.Join(errs...)
 }
 
+func attributesToUnsanitizedLabels(attrs pcommon.Map) labels {
+	ls := make(labels, attrs.Len())
+	attrs.Range(func(k string, v pcommon.Value) bool {
+		ls[k] = sanitizeUTF8(v.AsString())
```

Comment:
nit a little confusing to see this in a function named `ToUnsanitized`. Could you add doc comment explaining what this func does?

Reply:
added a comment